docs/_includes/migrate-from-msword.adoc

```diff
@@ -0,0 +1,224 @@
+// == Importing from MicroSoft Word
+:experimental:
+:url-pandoc: http://pandoc.org
+:url-ant: http://ant.apache.org/
+:url-google-asciidoc: https://chrome.google.com/webstore/detail/asciidoc-processor/eghlmnhjljbjodpeehjjcgfcjegcfbhk/
+:url-google-asciidoc-source:  https://github.com/Mogztter/asciidoc-googledocs-addon/
+
+=== Simple Pandoc
+
+Pandoc ({url-pandoc}) is a swiss army knife for converting file formats, and makes a good job of simple docx files.
+Use this command line:
+
+ $ pandoc --wrap=none --atx-headers --normalize --extract-media=build/images input.docx > output.adoc

```

Comment:
It looks like the command is not complete, input and output document types are missing.